cypress/e2e/formcheck.cy.js

```diff
@@ -0,0 +1,31 @@
+const userEmail = "jeanett.kestner@stud.noroff.no";
+const userPassword = "Kestner12";
+const incorrectEmail = "testing@noroff.no";
+const incorrectPassword = "passwordTest123";
+
+describe("Test for form validation and error handling", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.wait(1000);
+    cy.get("#registerForm button[type=button]").contains("Login").click();
+    cy.wait(1000);
+  });
+
+  it("Displays error for wrong password", () => {
+    cy.login(userEmail, incorrectPassword);
+  });
+
+  it("Displays error for wrong email", () => {
+    cy.login(incorrectEmail, userPassword);
+  });
+
+  it("Displays error for empty input field", () => {
+    cy.login("a", "a");
```

Comment:
Same here

Reply:
Added a part to the test code where it enters in the intended login credentials for wanted behaviour, and then checks for an error message aswell.